include/libdnf5/rpm/transaction_callbacks.hpp

```diff
@@ -74,27 +86,92 @@ class LIBDNF_API TransactionCallbacks {
     /// Called right before the rpm transaction is run
     /// @param total Number of elements in the rpm transaction
     virtual void before_begin(uint64_t total);
+
     /// Called after the transaction run finished
     /// @param success Whether the rpm transaction was completed successfully
     virtual void after_complete(bool success);
 
+    /// Report the package installation progress periodically.
+    /// @param item The TransactionPackage class instance for the package currently being installed
+    /// @param amount The portion of the package already installed
```

Comment:
Please add empty doc line after description. Do this for all methods.
Replace:
```
/// Report the package installation progress periodically.
/// @param item The TransactionPackage class instance for the package currently being installed
/// @param amount The portion of the package already installed

```
with:
```
/// Report the package installation progress periodically.
///
/// @param item The TransactionPackage class instance for the package currently being installed
/// @param amount The portion of the package already installed
```

The blank line is important for proper generation of documentation for Python.